package.json

```diff
@@ -81,12 +81,12 @@
   },
   "dependencies": {
     "classnames": "^2.2.0",
-    "cozy-ui": "^3.0.0-beta24",
+    "cozy-ui": "beta32",
```

Comment:
Didn't we find out this format causes issues with yarn's cache or something like that?